packages/form-js-viewer/test/spec/render/components/form-fields/Taglist.spec.js

```diff
@@ -667,8 +673,8 @@ describe('Taglist', function () {
 
         // when
         const filterInput = taglist.container.querySelector('.fjs-taglist-input');
-        fireEvent.focus(filterInput);
-        fireEvent.keyDown(filterInput, { key: 'Enter', code: 'Enter' });
+        await userEvent.click(filterInput);
+        await userEvent.keyboard('{enter}');
```

Comment:
I think enter is `capitalized`, could you check if it's really working or if it doesn't make any difference?

```suggestion
        await userEvent.keyboard('{Enter}');
```